go.mod

```diff
@@ -374,12 +374,6 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-replace k8s.io/client => github.com/kubernetes-client/go v0.0.0-20190928040339-c757968c4c36
-
-replace github.com/eclipse/paho.mqtt.golang => github.com/shivamkm07/paho.mqtt.golang v1.3.6-0.20220106130409-e28a1db639f8
```

Comment:
Don't remove this

Reply:
Why is this not upstream already? It is unacceptable to use @shivamkm07's personal fork for so many months. We need to find a different strategy.

Reply:
Latest version is 1.4.1 and we are stuck on this unmaintained 1.3.6 fork.

Reply:
Can you complete work on https://github.com/eclipse/paho.mqtt.golang/pull/578 @shivamkm07 ?

Reply:
@berndverst Updated the PR as requested. Waiting for the maintainers' response. Thanks!